src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -26,17 +29,19 @@ public class ModelManager implements Model {
 
     private FilteredList<Deck> filteredDecks;
     private Deck selectedDeck = null; // null when not selected, to switch to Optional<Deck> later on
+    private Review currReview = null;
```

Comment:
Small thing also but remember to switch to Optional in the future.

Reply:
Yep made an issue about it #82